Allow SYS_sched_getparam and SYS_sched_getscheduler
musl uses them for pthread_getschedparam()

--- a/src/3rdparty/chromium/sandbox/policy/linux/bpf_renderer_policy_linux.cc
+++ b/src/3rdparty/chromium/sandbox/policy/linux/bpf_renderer_policy_linux.cc
@@ -94,6 +94,11 @@
     case __NR_pwrite64:
     case __NR_sched_get_priority_max:
     case __NR_sched_get_priority_min:
+#ifndef __GLIBC__
+    case __NR_sched_getparam:
+    case __NR_sched_getscheduler:
+    case __NR_sched_setscheduler:
+#endif
     case __NR_sysinfo:
     case __NR_times:
     case __NR_uname:
